lib/validator/validator.go

```diff
@@ -165,8 +165,8 @@ func validateTag(v *validator.Validate, topStruct reflect.Value, currentStructOr
 func validateLabels(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
 	l := field.Interface().(map[string]string)
 	log.Debugf("Validate labels: %s", l)
-	for k, v := range l {
-		if !labelRegex.MatchString(k) || !labelRegex.MatchString(v) {
+	for k := range l {
+		if !labelRegex.MatchString(k) {
```

Comment:
Speaking to Shaun, I don't think values need validating - although we should probably limit their length to match the length limit used by k8s.